.github/workflows/mobile-traffic_director.yml

```diff
@@ -0,0 +1,48 @@
+name: mobile_traffic_director
+
+on:
+  schedule:
+  # Once a day at midnight.
+  - cron: '0 0 * * *'
+  # Allows manual triggering in the UI. Makes it easier to test.
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.head_ref-github.workflow || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  cc_test:
+    if: |
+      ${{
+          github.repository == 'envoyproxy/envoy'
+          && (github.event.schedule
+              || !contains(github.actor, '[bot]'))
+      }}
+    name: cc_test
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        # We must fetch at least the immediate parents so that if this is
+        # a pull request then we can checkout the head.
+        fetch-depth: 2
```

Comment:
is this necessary? im guessing this can just use the default `fetch-depth = 0`

Reply:
you're right, it was only needed when `on` contained `pull_request` as a trigger.  I've updated it to `0`.